stempy/reader.h

```diff
@@ -569,7 +582,7 @@ std::future<void> SectorStreamMultiPassThreadedReader::readAll(Functor& func)
   }
 
   // Reset counter
-  m_processed = { 0 };
+  m_processed = 0;
```

Comment:
Why this change?

Reply:
It fixed lots of warnings on Mac. Hopefully it will work at runtime...